workers/ohsome_quality_analyst/cli/cli.py

```diff
@@ -82,7 +83,15 @@ def list_fid_fields():
 def get_available_regions():
     """List available regions."""
     regions = asyncio.run(db_client.get_available_regions())
```

Comment:
Since PR #183 is merged using the function `get_regions()` to get a list of  (name, id) tuples would be more efficient.